lib/questions.rb

```diff
@@ -1,61 +1,82 @@
 # keep only the elements that start with an a
 def select_elements_starting_with_a(array)
+  array.keep_if {|array| array =~ /\A[a]/}
 end
 
 # keep only the elements that start with a vowel
 def select_elements_starting_with_vowel(array)
+  array.keep_if {|array| array =~ /\A[aeiou]/}
```

Comment:
Shadowing outer local variable - `array`.<br>Space between { and | missing.<br>Space missing inside }.